server/configuration.js

```diff
@@ -1,25 +1,33 @@
 export const setEnvironment = () => {
     const productionGraphqlEndpoint = 'https://tray.io/graphql';
     const stagingGraphqlEndpoint = 'https://staging.tray.io/graphql';
+    const frontendStagingGraphqlEndpoint = 'https://frontend-staging.tray.io/graphql';
 
-    const appUrlStaging = 'https://embedded.staging.tray.io';
     const appUrlProd = 'https://embedded.tray.io';
+    const appUrlStaging = 'https://embedded.staging.tray.io';
+    const appUrlFrontendStaging = 'https://embedded.frontend-staging.tray.io';
```

Comment:
you sure this doesn't need `/graphql` in the end ? since it should be the public graphql endpoint